doc/sphinx/source/utils/RTW/user_guide/quick_start.rst

```diff
@@ -38,5 +38,17 @@ Quick Start Guide
         export CYLC_VERSION=8
         cylc vip -O jasmin
 
-* Browse the logs using `Cylc Review`_, a web service for browsing logs via an
-  HTTP interface.
+  * on DKRZ:
+
+    * add the following line to your ``~/.bashrc`` file to ensure the Cylc and
+      Rose executables can be found::
+
+        export PATH=/work/bd0854/metomi/bin:$PATH
+
+    * run the RTW on DKRZ::
+
+        cd ESMValTool/esmvaltool/utils/recipe_test_workflow
+        cylc vip -O dkrz
```

Comment:
Here are instructions on how to run the RTW on DKRZ. Rendered version of these instructions are at https://esmvaltool--3837.org.readthedocs.build/en/3837/utils/RTW/user_guide/quick_start.html. Note that some of the process tasks are failing, I assume due to development changes since the last release. I will open new issues for those next week 👍